dev-util/trace-cmd/trace-cmd-2.7.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=(python2_7)
+
+inherit eutils linux-info python-single-r1 toolchain-funcs
+
+DESCRIPTION="User-space front-end for Ftrace"
+HOMEPAGE="https://git.kernel.org/cgit/linux/kernel/git/rostedt/trace-cmd.git"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://git.kernel.org/pub/scm/linux/kernel/git/rostedt/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://git.kernel.org/pub/scm/linux/kernel/git/rostedt/trace-cmd.git/snapshot/${PN}-v${PV}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-v${PV}"
+fi
+
+LICENSE="GPL-2 LGPL-2.1"
```

Comment:
The Licence Text for both Licences mentioned ```or (at your option) any later version```, so i think this should be ```GPL-2+ LGPL-2.1+```
Same applies for the ```9999``` ebuild

Reply:
Thanks for noticing, I've updated this PR accordingly.